ui/lib/core/addon/components/toolbar-link.js

```diff
@@ -8,14 +8,18 @@ import Component from '@glimmer/component';
  * ```js
  * <Toolbar>
  *   <ToolbarActions>
- *     <ToolbarLink @params={{array 'vault.cluster.policies.create'}} @type="add" @disabled={{true}} @disabledTooltip="This link is disabled">
+ *     <ToolbarLink @route="vault.cluster.policies.create" @type="add" @disabled={{true}} @disabledTooltip="This link is disabled">
  *       Create policy
  *     </ToolbarLink>
  *   </ToolbarActions>
  * </Toolbar>
  * ```
  *
- * @param {array} params - Array to pass to LinkTo
+ * @param {string} route - route to pass to LinkTo
+ * @param {Model} model - model to pass to LinkTo
+ * @param {Array} models - array of models to pass to LinkTo
+ * @param {Object} query - query params to pass to LinkTo
+ * @param {boolean} replace - replace arg to pass to LinkTo
```

Comment:
I don't see this passed to `<LinkTo>` in the hbs file above...am I'm just missing it?

Reply:
Ah yes good catch. I noticed that there were instances of `ToolbarLink` where `replace` was being passed in but it wasn't actually supported. Looks like I added the documentation only :grin:. I've added it to the `LinkTo` components.